docs/metrics/index.md

```diff
@@ -0,0 +1,132 @@
+[👈 Back to README](../../README.md)
+
+# Overview
+
+Test Pilot loves metrics.  Our whole purpose is to ask questions, measure and
+analyaze responses, and make recommendations.  To that end, we have a few tools
+we use:
+
+## Google Analytics
+
+We build on a Google Analytics foundation for sheer breadth of built-in
+functionality.  GA collects a lot of data for "free" which helps answer
+unknown-unknowns when building rapid prototypes, and having real-time results
+is just gravy.  We use GA for all basic measuring, funnel tracking,
+click-through rates, and simple A/B testing.
+
+## Telemetry
```

Comment:
We should probably add some context about what [capital-T Telemetry means](https://wiki.mozilla.org/Telemetry/FAQ#What_is_Telemetry_and_why_is_it_good_for_Firefox_users.3F).